tests/modules/baseline_tests.rs

```diff
@@ -43,10 +43,7 @@ fn record_local_baseline(font: &Font, name: &str, character_index: u16, size: f3
     if !Path::new(&reference_path).exists() || fs::read(reference_path).unwrap() != encoded {
         // only write out local bitmap when it doesn't match (this saves a lot of disk i/o time)
         fs::create_dir_all(Path::new(&local_path).parent()?).ok();
-        let err = fs::write(&local_path, encoded);
-        if err.is_err() {
-            assert!(false, "Unable to save file {}: {}", &local_path, err.unwrap_err());
-        }
+        fs::write(&local_path, encoded).unwrap();
```

Comment:
This is less verbose than the older code.

Reply:
Oh, I see. I assume you prefer it this way instead:

```rust
if let Err(err) = err {
    panic!("Unable to save file {}: {}", &local_path, err);
}
```

Reply:
Sure

Reply:
Alrighty, all should be good now. 👍